llvm/lib/Target/X86/X86ISelLowering.cpp

```diff
@@ -3405,6 +3405,9 @@ X86TargetLowering::getJumpConditionMergingParams(Instruction::BinaryOps Opc,
                                                  const Value *Rhs) const {
   using namespace llvm::PatternMatch;
   int BaseCost = BrMergingBaseCostThresh.getValue();
+  // With CCMP, branches can be merged in a more efficient way.
+  if (BaseCost >=0 && Subtarget.hasCCMP())
+    BaseCost += 6;
```

Comment:
Is this based on any benchmarks or the case you commented below?

Reply:
It's not tuned yet. I just pick a minimum value to make the test pass.

Reply:
While I imagine with APX the value should be higher, can you just put the necessary tuning value in the test command line?

Reply:
We can update the value after tuning. No, I need to generate CCMP/CTEST w/o adding tuning flag, that's the use case.

Reply:
hmm? This flag is just to keep basic-blocks together. Why can't it be part of test commandline? (i.e the `RUN` line).

Reply:
It can be part of test command line.  My understanding is that you suggested removing `BaseCost += 6` here and **only** putting the flag in the test. 

I'd like the user to generate the CCMP/CTEST with front-end flag `-mapx-features=nf` for the cases in the test, w/o the tuning flags `x86-*cost`.

Reply:
Done.  Added new flag "-x86-br-merging-ccmp-bias". We can increase its value in command line if needed.